pimd/pim_zebra.c

```diff
@@ -141,20 +141,6 @@ static int pim_zebra_if_address_add(ZAPI_CALLBACK_ARGS)
 #endif
 	}
 
-	if (!CHECK_FLAG(c->flags, ZEBRA_IFA_SECONDARY)) {
```

Comment:
Doesn't this cause pim to loose all knowledge of secondary addresses?  We need to keep that knowledge in there imo.  I do not believe this is the right solution here